### ISSUE TITLE
Feature Request: Increment and Decrement operators

### ISSUE THREAD

**Issue by zauberparacelsus** [MyWorldLLC/Chipmunk#30]
This is a feature request for the ++ and -- operators found in C and C-like languages.

**Comment by danielperano** [comment]
These are partially implemented - the `++` and `--` postfix operators are implemented and increment/decrement by 1 respectively, but they do not re-assign the variable, they only return the incremented value. They should behave as in the C family of languages, however.

**Comment by zauberparacelsus** [comment]
> They should behave as in the C family of languages, however.

Including the part where they change depending on whether they are put before or after the variable?

**Comment by danielperano** [comment]
Yes - that's their behavior elsewhere so they should behave the same in Chipmunk.
